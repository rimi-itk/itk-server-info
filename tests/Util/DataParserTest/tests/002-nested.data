<server>
<php>
<version><![CDATA[PHP
]]></version>
<modules><![CDATA[Modules
]]></modules>
</php>
</server>


