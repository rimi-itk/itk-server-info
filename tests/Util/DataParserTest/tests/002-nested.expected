php:
  version: |
    PHP
  modules: |
    Modules
